src/components/SigningModals/MultiScreenModal.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Modal,
+  Viewport,
+  springs,
+  textStyle,
+  useViewport,
+  GU,
+} from '@aragon/ui'
+import { Spring, Transition, animated } from 'react-spring'
+import { useSteps } from '../../hooks'
+
+const AnimatedDiv = animated.div
+
+const DEFAULT_MODAL_WIDTH = 700
```

Comment:
May have to quickly double check with @bpierre; this is a situation where I'm not sure if a GU would make sense (aragonUI's breakpoints are in both 🙈).